src/qt/sendcoinsdialog.cpp

```diff
@@ -628,7 +628,7 @@ void SendCoinsDialog::updateSmartFeeLabel()
     if(!model || !model->getOptionsModel())
         return;
 
-    int nBlocksToConfirm = (int)25 - (int)std::max(0, std::min(24, ui->sliderSmartFee->value()));
+    int nBlocksToConfirm = (int)defaultConfirmTarget - (int)std::max(ui->sliderSmartFee->minimum(), std::min(ui->sliderSmartFee->maximum(), ui->sliderSmartFee->value()));
```

Comment:
So this would become ``nTxConfirmTarget = defaultConfirmTarget - ui->sliderSmartFee->value();``, right?